<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFinanceiroContas</class>
 <widget class="QWidget" name="WidgetFinanceiroContas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableVencidos">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="TableView" name="tableVencer">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Vencer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Vencidos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="labelContasReceberBuscar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Buscar:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditBusca">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>183</width>
            <height>975</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBoxFiltros">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="title">
              <string>Filtros:</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="16" column="0">
               <widget class="QPushButton" name="pushButtonAdiantarRecebimento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Antecipar recebimento</string>
                </property>
               </widget>
              </item>
              <item row="17" column="0">
               <widget class="QPushButton" name="pushButtonInserirLancamento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Inserir lançamento</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QGroupBox" name="groupBoxRealizado">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Realizado De/Até</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_8">
                 <item row="1" column="0">
                  <widget class="QDateEdit" name="dateEditRealizadoDe">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QDateEdit" name="dateEditRealizadoAte">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButtonTodos">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Todos</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="15" column="0">
               <widget class="QPushButton" name="pushButtonImportarFolhaPag">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Importar folha pag.</string>
                </property>
               </widget>
              </item>
              <item row="13" column="0">
               <spacer name="verticalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>290</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="0">
               <widget class="QRadioButton" name="radioButtonPago">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Pago</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QRadioButton" name="radioButtonRecebido">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Recebido</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QGroupBox" name="groupBoxLojas">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Loja</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="ItemBox" name="itemBoxLojas">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radioButtonPendente">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Pendente</string>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QGroupBox" name="groupBox">
                <property name="title">
                 <string>R$ De/Até</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxDe">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="prefix">
                    <string>R$ </string>
                   </property>
                   <property name="maximum">
                    <double>99999.990000000005239</double>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxAte">
                   <property name="showGroupSeparator" stdset="0">
                    <bool>true</bool>
                   </property>
                   <property name="prefix">
                    <string>R$ </string>
                   </property>
                   <property name="maximum">
                    <double>99999.990000000005239</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QGroupBox" name="groupBoxVencimento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="title">
                 <string>Vencimento De/Até</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="1" column="0">
                  <widget class="QDateEdit" name="dateEditVencimentoDe">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QDateEdit" name="dateEditVencimentoAte">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="calendarPopup">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QRadioButton" name="radioButtonCancelado">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Cancelado</string>
                </property>
               </widget>
              </item>
              <item row="18" column="0">
               <widget class="QPushButton" name="pushButtonInserirTransferencia">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Inserir transferência</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="radioButtonAgendado">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Agendado</string>
                </property>
               </widget>
              </item>
              <item row="20" column="0">
               <widget class="QPushButton" name="pushButtonReverterPagamento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Reverter pagamento</string>
                </property>
               </widget>
              </item>
              <item row="14" column="0">
               <widget class="QPushButton" name="pushButtonRemessaItau">
                <property name="text">
                 <string>Gerar remessa ITAÚ</string>
                </property>
               </widget>
              </item>
              <item row="19" column="0">
               <widget class="QPushButton" name="pushButtonExcluirLancamento">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Excluir lançamento</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="radioButtonConferido">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Conferido</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="TableView" name="table">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableVencidos</tabstop>
  <tabstop>tableVencer</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>radioButtonTodos</tabstop>
  <tabstop>radioButtonPendente</tabstop>
  <tabstop>radioButtonAgendado</tabstop>
  <tabstop>radioButtonRecebido</tabstop>
  <tabstop>radioButtonPago</tabstop>
  <tabstop>radioButtonCancelado</tabstop>
  <tabstop>groupBoxLojas</tabstop>
  <tabstop>itemBoxLojas</tabstop>
  <tabstop>groupBoxVencimento</tabstop>
  <tabstop>dateEditVencimentoDe</tabstop>
  <tabstop>dateEditVencimentoAte</tabstop>
  <tabstop>doubleSpinBoxDe</tabstop>
  <tabstop>doubleSpinBoxAte</tabstop>
  <tabstop>pushButtonRemessaItau</tabstop>
  <tabstop>pushButtonImportarFolhaPag</tabstop>
  <tabstop>pushButtonAdiantarRecebimento</tabstop>
  <tabstop>pushButtonInserirLancamento</tabstop>
  <tabstop>pushButtonInserirTransferencia</tabstop>
  <tabstop>pushButtonExcluirLancamento</tabstop>
  <tabstop>pushButtonReverterPagamento</tabstop>
  <tabstop>table</tabstop>
  <tabstop>lineEditBusca</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
